src/mol-model/structure/query/queries/modifiers.ts

```diff
@@ -576,16 +584,14 @@ export function surroundingLigands({ query, radius, includeWater }: SurroundingL
     };
 }
 
-const _entity_type = StructureProperties.entity.type;
-function testIsWater(l: StructureElement.Location) {
-    return _entity_type(l) === 'water';
-}
-
 function getPrdAsymIdx(structure: Structure) {
     const model = structure.models[0];
     const ids = new Set<string>();
-    if (!MmcifFormat.is(model.sourceData)) return ids;
-    const { _rowCount, asym_id } = model.sourceData.data.db.pdbx_molecule;
+
+    // Need to do this menually to prevent a cyclical import causing
```

Comment:
How about splitting src\mol-model-formats\structure\mmcif.ts into mmcif/format.ts and mmcif/data.ts or so? The cyclic dependency issue can creep up over and over again and thios would fix it.

Reply:
In general, these are like 200 places where we do cyclic imports. It only causes an issue where there is the inline code execution though.

Reply:
Another solution would be to move where we do the format provider registration to the files with the providers.

Reply:
Don't know how to change this. I tried splitting the inline code execution into a separate file and importing it from the `mmcif.ts`, but that didn't work.

I think overall we will be fine and if this creeps up again, we do a similar fix. But perhaps I am missing something obvious...